run/.gitignore

```diff
@@ -8,6 +8,8 @@
 node_modules
 
 # Generated
+lib/cjs
+lib/es6
 lib/
```

Comment:
It seems to me that @cycle/run is the only exception, where we have this backwards-compatible `lib` location. So don't ignore this.